UCSLibraries/src/main/java/it/cnr/iit/ucs/exceptions/EndAccessException.java

```diff
@@ -0,0 +1,14 @@
+package it.cnr.iit.ucs.exceptions;
+
+public class EndAccessException extends Exception {
```

Comment:
I would use a single StatusException for any ucs status problem

Reply:
I prefer having different exception mostly because the reason can be different. But for sure we can prepare also a generic one named StatusException and then all the others will extend that one. What you think?

Reply:
yes that could be useful to catch any statusexception. there's already a StatusException